pagopa/pn-frontend
# Feature/pn 2412

## Short description
i delete default value on pf in selection of radio-button and delete "disabled" attribute in pg radio-button. Also do it refactoring for layout passed from grid to stack

## List of changes proposed in this pull request
- delete default value on pf in selection of radio-button 
- delete "disabled" attribute in pg radio-button

## How to test
now is possible check radio-button to select Persona giuridica in create mandete
